charlib/characterizer/procedures/combinational/delay.py

```
@@ -179,13 +186,14 @@ def measure_delays_for_path_with_criterion(cell, config, settings, variation, pa
 
         # Build LUT
         delay_measurements =[analysis.measurements[name] for analysis in analyses.values() if name in analysis.measurements]
-        delay = criterion(delay_measurements) @ PySpice.Unit.u_s
+        delay = (criterion(delay_measurements) if delay_measurements else -1) @ PySpice.Unit.u_s
         lut_name, meas_path = name.split('__')
```

Ruff (0.15.21)

[warning] Unpacked variable `meas_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)